python-package/lightgbm/dask.py

```diff
@@ -107,6 +107,22 @@ def _find_ports_for_workers(client: Client, worker_addresses: Iterable[str], loc
     return worker_ip_to_port
 
 
+def _find_random_open_port():
```

Comment:
we've been trying to use type hints everywhere in the Dask module, in a step towards #3756 

can you please add this return hint?

```suggestion
def _find_random_open_port() -> int:
```